### PR TITLE
(argusmonitor) New package

### PR DESCRIPTION
**Pull request by p3lim**
## Description
Adds Argus Monitor.
https://www.argotronic.com/en/

## Motivation and Context
Package doesn't exist.

## How Has this Been Tested?

Ran `update.ps1`, `Test-Package -Install` and `Test-Package -Uninstall`.  

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Comment by p3lim** [comment]
- The uninstaller opens a web page upon completion, couldn't find any way to prevent that.
- They have no "public" license, so the one in the repo was pulled from the installer.

**Comment by p3lim** [comment]
Also having some trouble getting these options enabled, they are all unselected when running unattended:
611a 1

File: automatic/argusmonitor/argusmonitor.nuspec [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <authors>Argotronic UG (haftungsbeschränkt)</authors>
+    <projectUrl>https://www.argotronic.com/en/</projectUrl>
+    <iconUrl>http://help.argusmonitor.com/lib/h12.png</iconUrl>
+    <copyright>Copyright © 2008-2017 Argotronic UG (haftungsbeschränkt)</copyright>
+    <licenseUrl>https://www.argotronic.com/en/gtc.pdf</licenseUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <projectSourceUrl>https://www.argotronic.com/en/</projectSourceUrl>
+    <docsUrl>http://help.argusmonitor.com/</docsUrl>
+    <bugTrackerUrl>https://www.argotronic.com/en/contact.php</bugTrackerUrl>
+    <tags>argusmonitor argus monitor system fan temperature control admin</tags>
```
Comment (majkinetor) [inline automatic/argusmonitor/argusmonitor.nuspec]: Needs `trial`

**Comment by majkinetor** [comment]
You can use AHK to click those options if not exposed any other way.

BTW, if you use this tool, can you tell if it remembers previous sessions histogram after the computer restarts ?

**Comment by p3lim** [comment]
> You can use AHK to click those options if not exposed any other way.

I've contacted their support about silent options, they are usually very willing to add features.

> BTW, if you use this tool, can you tell if it remembers previous sessions histogram after the computer restarts ?

It does not, sadly.

**Comment by AdmiringWorm** [comment]
Seems the package doesn't exist, so no choco user needs to be added.

**Comment by p3lim** [comment]
The package was not ready to be released yet, I have not heard back from Argotronic or used AHK to circumvent the install options. Currently the software does not install correctly because of missing options during installation.

**Comment by AdmiringWorm** [comment]
@p3lim running the installer is usually enough, if the installer doesn't create shortcuts or allow options that's usually okay (as long as that is the default). Users could always run the installer in non-silent mode, also since it's a NSIS installer, parameters/options usually isn't available to be used.

In any case, we can always push out a fix version such options is available.

**Comment by p3lim** [comment]
The default options present in the installer when not run silently differs from the silent ones.
In the screenshot I attached above, the options with the green checkboxes are not handled in silent mode.

It's just shortcuts and autostart, but it might be confusing to users why the program does not have them by default after installing.
If/when I get a response from Argotronic I'll submit a patch, but I was not expecting the PR to get merged before that happened s'all.

**Comment by AdmiringWorm** [comment]
I see, unfortunately we can't know if you don't want the PR to be merged unless you tell us exactly that.

While you did mention that you had trouble with the options, this wasn't something we needed to hold up the PR for.

BTW, the package haven't yet been approved anyhow so we can always just plainly reject the package until a finalised version is available if you want.

For future notices though, if there is a PR that you don't want to be merge when we feel it qualifies for this repo, just prefix the title of the PR with `[WIP]`, and when it's done just remove it from the title.
That would cause us to skip over the PR until we notice it doesn't have WIP anymore in the title.

**Comment by p3lim** [comment]
> BTW, the package haven't yet been approved anyhow so we can always just plainly reject the package until a finalised version is available if you want.

That would probably be best.

> For future notices though, if there is a PR that you don't want to be merge when we feel it qualifies for this repo, just prefix the title of the PR with [WIP], and when it's done just remove it from the title.
That would cause us to skip over the PR until we notice it doesn't have WIP anymore in the title.

Noted.

**Comment by AdmiringWorm** [comment]
@p3lim done.
Btw, what is your username on chocolatey?
I'll get you added as a maintainer to the package as well (it's preferred to have at least one main maintainer other than chocolatey on packages).

**Comment by AdmiringWorm** [comment]
I've also renamed the update.ps1 file so it won't be automatically updated until it's ready.

When you hear back from the developers, and open a new PR just rename the `_update.ps1` file back to `update.ps1` along with the rest of the logic you want.

**Comment by p3lim** [comment]
> Btw, what is your username on chocolatey?

Same as here.

> When you hear back from the developers, and open a new PR just rename the _update.ps1 file back to update.ps1 along with the rest of the logic you want.

Alright, thanks for the assistance.

**Pull request merged**